tests/test_futures.py

```diff
@@ -80,6 +84,10 @@ def test_datacard_2l_ht():
         "python",
         "analysis/topEFT/datacard_maker.py",
         "analysis/topEFT/histos/output_check_yields.pkl.gz",
+        "--var-lst",
+        "njets",
+        "ht",
+        "ptbl",
         "-j",
         "9"
```

Comment:
It wont make a difference, but we could also just use e.g. `HT` here (since it is the check `HT` function) and change 9 to 0 (since 9 was originally `HT 2lss_p`. Feel free to keep it how it is now if figuring out the `-j` numbers is too much of a headache.

Reply:
Thanks, if it's ok with you I'd probably prefer to just leave it as it is so we don't have to figure out the `-j` numbers.

Reply:
Sounds good.